src/style.css

```
@@ -0,0 +1,60 @@
+body {
+  margin: 0;
+  font-family: Roboto;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  height: 40px;
+}
+
+.menu {
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.menu_item {
+  margin-left: 20px;
+}
+
+.menu_item:first-child {
+  margin-left: 0;
```

Instead two these selectors you can create only one.
```suggestion
  .menu_item:not(:first-child) {
    margin-left: 20px;
  }
```